lib/bulkrax/rerun_errored_entries_for_importer_service.rb

```diff
@@ -0,0 +1,146 @@
+module Bulkrax
+  ##
+  # This class is responsible for rescheduling all failed importers; and creating a run state as if
+  # we had run everything.
+  #
+  # @note This class is starting it's life in the Adventist repository but is being designed so that
+  #       we could copy it rather easily into Bulkrax.
+  #
+  # @see https://github.com/scientist-softserv/adventist-dl/issues/430
+  #
+  # @example
+  #   service = Bulkrax::RerunErroredEntriesForImporterService.new(importer_id: 123, last_run_id: 456)
+  #   service.call
+  class RerunErroredEntriesForImporterService
+
+    def self.call(importer_id:, last_run_id: nil, error_class: [], logger: Rails.logger)
+      new(importer_id: importer_id, last_run_id: last_run_id, error_classes: error_class, logger: logger)
+    end
+
+    ##
+    # @param importer_id [Integer]
+    # @param last_run_id [Integer]
+    # @param error_classes [String] only re-run for entries that had failures matching the given
+    #        :error_classes.
+    # @param logger [Logger]
+    def intialize(importer_id:, last_run_id: nil, error_classes: [], logger: Rails.logger)
+      @logger = logger
+      @importer = Bulkrax::Importer.find(importer_id)
+      if last_run_id
+        @last_run = @importer.importer_runs.find(last_run_id)
+      else
+        @last_run = @importer.last_run
+      end
+      @new_run = create_new_run_from_last_run
+      @error_classes = Array.wrap(error_class).map(&:to_s)
```

Comment:
```suggestion
      @error_classes = Array.wrap(error_classes).map(&:to_s)
```

should error class be error_classes ?